examples/rich-text/webpack.config.js

```diff
@@ -0,0 +1,21 @@
+const path = require('path');
+
+module.exports = {
+  entry: './client.js',
+  output: {
+    path: path.resolve(__dirname, 'static/dist'),
+    filename: 'bundle.js'
+  },
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+      }
+    ]
+  },
```

Comment:
As I [said before](https://github.com/share/sharedb/pull/683/files/e845f8761d50d5912a0d7bdc81ede1f3c79b1946#diff-4001198f2b1552f7c331a4c754f8b96bc85fa9ace87590061e8f05a5309f82a3), this isn't needed.

Please remove.